Makefile

```diff
@@ -13,7 +13,7 @@ format:
 
 .PHONY: test
 test:
-	python3 -m tox -p auto
+	python3 -m tox
```

Comment:
Since we're running `tox` directly instead of `make test` in Github actions, revert this to run tests in parallel locally?

Reply:
Then the output is garbled again. Or if we run in parallel without `-o`, warning messages are swallowed, and you can't use `breakpoint` for debugging tests, for instance.

Reply:
I don't have strong opinions here, as I usually just run `tox` directly myself. Often just running one env, ie `tox -e py`.

Reply:
Remved this change, it's not related to the rest of the changes in this PR :ok_person:

Reply:
I'm fine with adding `-o` here for improved output - I think we're doing that in all the other projects too 👍